### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by mansipadgelwar**
Home page

- Users can see a landing page with a list of a few categories.
- Clicked on any one of the categories it will be redirected to the video list page with the selected category.

### CODE REVIEW

**Review by abhij1607 — APPROVED** [review]
Neat & Clean code
Good folder structure

Can be merged

File: src/components/CategoryCard/CategoryCard.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    position: absolute;
+    inset: 0;
+    font-weight: bolder ;
+    font-size: 1.3rem;
+    color: var(--white);
+    text-shadow:
+    0 0 7px #fff,
+    0 0 10px #fff,
+    0 0 21px #fff;
+    background: rgba(0,0,0,0.4);
```
Comment (pradipgharge) [inline src/components/CategoryCard/CategoryCard.css]: use color variable here

File: src/components/CategoryCard/CategoryCard.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    letter-spacing: 0.2rem;
+    cursor: pointer;
+  }
+
+  .category-card-container .video-heading:hover{
+      font-size: 1.5rem;
+  }
+    
+  .video-thumbnail {
+    max-height: 200px;
```
Comment (pradipgharge) [inline src/components/CategoryCard/CategoryCard.css]: use rems instead of px

File: src/pages/Home/Home.jsx [hunk 3]
```diff
@@ -1,5 +1,10 @@
 import "../Home/Home.css";
-import { Sidebar, VideoCard } from "../../components";
+import { Sidebar, CategoryCard } from "../../components";
+import Ballroom from "../../assets/ballroom.jpg";
+import Contemprary from "../../assets/contemprary.jpg";
+import HipHop from "../../assets/hip-hop.jpg";
+import Banner from "../../assets/banner.jpg";
```
Comment (pradipgharge) [inline src/pages/Home/Home.jsx]: you can make an index.js file in the assets folder to store all images and then you can easily import them with one line of code anywhere .

**Review by pradipgharge — APPROVED** [review]
Suggested some changes, rest looks good to merge !

**Pull request merged**